I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 74, in script
    jump next
ScriptError: could not find label 'next'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 74, in script
    jump next
  File "C:\Users\uSER\renpy-8.1.1-sdk\renpy\ast.py", line 1992, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\uSER\renpy-8.1.1-sdk\renpy\script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'next'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.1.23060707
kaistmossolescape 1.0
Sat Jul 15 15:37:13 2023
